src/rt-app_parse_config.c

```diff
@@ -811,7 +811,7 @@ parse_thread_data(char *name, struct json_object *obj, int index,
 		data->phases = malloc(sizeof(phase_data_t) * data->nphases);
 		parse_thread_phase_data(obj,  &data->phases[0], opts, (long)data);
 		/* Get loop number */
-		data->loop = -1;
+		data->loop = get_int_value_from(obj, "loop", TRUE, -1);
 	}
```

Comment:
This is called when there is no "phase" object defined in the "thread" object. In this case, we assume that events and loop of the phase are put directly in the "thread".
In this case we have only 1 "loop" key value for phase and thread and as we call parse_thread_phase_data with thread object, the "loop" will already be taken in the phase.
With this change, the "loop" key will be used twice. Once in the parse_thread_phase_data and once in parse_thread_data.

so if we have
"thread0" : {
			"instance" : 1,
			"loop" : 2,
			"run" :   10000,
}

we will run 4 time the run events